### DISCUSSION TITLE
Error Handling?

### DISCUSSION THREAD

**Discussion by cstadther** [magicbell/community#182] [Questions] [answered]
I'm currently getting a lot of POST errors and timeouts with the magicbell-react component version(10.10.2)  The problem is these bubble up my global "unhandled errors" function.  Wrapping the react component in an error boundary does not stop this behavior.  Is there a way I can swallow all the errors generated by the magicbell-react component?

![image](https://github.com/magicbell-io/community/assets/5796176/e3287914-97ef-4069-b962-080c7a82673d)
![image](https://github.com/magicbell-io/community/assets/5796176/066b8291-18ad-4aeb-b3f0-1a9970c66d6b)

**Comment by unamashana** [comment, marked as answer]
TLDR; This error is safe to ignore and we will fix this in an upcoming release. It's a case of rolling out an early API and using the wrong HTTP code to handle a common case. 

## More details

That error is coming from your inbox integration. We're fetching the theme that you might have configured in our dashboard. The error message should be clearer, but its safe to ignore. There are two options to make it not error though:

1) Configure a theme in our dashboard. To do so, go to your project > settings > inbox. Then customize, and click "publish theme".
2) Provide a `theme`, `locale`, and `images` property to the sdk. This works, because we don't fetch the theme config when a config is provided in code. If `images` property is missing you can provide something like the below to prevent the network request from happening.

{
locale: 'en',
theme: { /* your theme */ },
images: { emptyInboxUrl: 'https://assets.magicbell.io/images/empty_inbox.png' },
}
